HTML/html_completions.py

```diff
@@ -199,7 +199,7 @@ def __init__(self):
 
     def on_query_completions(self, view, prefix, locations):
         # Only trigger within HTML
-        if not view.match_selector(locations[0], "text.html - source - string.quoted"):
+        if not view.match_selector(locations[0], "text.html - source - string.quoted - meta.tag.style punctuation.definition.tag.begin"):
```

Comment:
Note: this has the side effect of showing buffer completions when typing a new tag immediately before the `<style type="text/css">` i.e. type `<`. I'd welcome any ideas for combating this. I am thinking to have a separate scope for `meta.tag.style.begin.html` and `meta.tag.style.end.html`, as per https://github.com/sublimehq/Packages/issues/442.